frontend/package.json

```diff
@@ -31,6 +31,7 @@
     "react-confirm-alert": "^3.0.6",
     "react-dom": "^18.0.0",
     "react-hook-form": "^7.29.0",
+    "react-hotkeys-hook": "^4.3.8",
```

Comment:
7.4Kb gzipped minifed. Not too small
https://bundlephobia.com/package/react-hotkeys-hook@4.3.8

Reply:
but the codebase looks better.